### DISCUSSION TITLE
Mirth "Cloud" Connect in AWS - Admin Control Center

### DISCUSSION THREAD

**Discussion by twest-mirthconnect** [nextgenhealthcare/connect#4668] [Mirth Mockups]
Alrighty, folks, we have another mock-up... this is very early in the interaction stage but I am working with the team on trying to kill two birds with one stone:

1. Provide Mirth Connect "as-is" as a service in Amazon's Web Services (more to come on this)
2. Merge some of the functionality that is currently operating in the Mirth Connect Appliance tech (mainly the front end or control panel with a bunch of gadgets)

Below you will see just a very simple view of a Mirth Cloud Connect front end that users would see and take advantage in a variety of ways. 

The idea will be to provide a web page that can be customized to your liking and that we can prioritize the type of mission-critical features that will help make your job, project, whatever easier and more manageable. After you had a look, I would appreciate it if you would share what widgets, gadgets,  and kinds of features you would like to see ready-made and easy to use.

As always, all feedback is welcome and wanted. :)

Thanks,
Travis

![Control Panel for Mirth Connect in AWS](https://user-images.githubusercontent.com/87716758/128562264-8d6d19b2-b278-4fe1-849f-ff3cbcee39bb.png)
![Menu](https://user-images.githubusercontent.com/87716758/128562280-9a54c6f8-d608-407e-8d06-071b0613810e.png)
![Add Widget](https://user-images.githubusercontent.com/87716758/128562287-931d0ef1-1bb0-4951-a17b-089d35f0cf02.png)

**Comment by twest-mirthconnect** [comment]
wow... everybody is either stunned at my amazing use of wire frames OR... they love my use of color. :) Should I have changed that to green through... decisions decisions.

**Reply by pacmano1** [reply]
Well.  I think you should not mix unrelated concepts.  AWS or Azure or self hosted infrastructure all need appropriate instrumentation and monitoring.  If this feature address outstanding features I'd share those tiks.  Your screenshots seem a bit of "build it and they will come". It's a bit of scatological approach.

**Reply by twest-mirthconnect** [reply]
Thanks for responding... hope it didn't come off too cheeky. :/ I appreciate the quick note.
